src/components/CreateCard.jsx

```diff
@@ -140,46 +142,38 @@ class CreateCard extends Component {
 
           <div className="flashcards">
             <div className="flashcard">
-              <div className="content">
-                <strong>front text</strong>
-                <textarea
-                  name="frontText"
-                  id="frontText"
-                  className="text"
-                  placeholder="Please press the record button"
-                  onChange={this.handleFrontTextChange}
-                  value={this.props.frontText}
-                />
-                <div
-                  title="record new flashcard"
-                  className="microphone"
-                  id="microphone-learning-1"
-                  data-id="1"
-                  data-linked-to="frontText"
-                  onClick={this.handleRecordClick}
-                />
-              </div>
+              <strong>front text</strong>
+              <textarea
+                name="frontText"
+                id="frontText"
+                className="text"
+                placeholder="Please press the record button"
+                onChange={this.handleFrontTextChange}
+                value={this.props.frontText}
+              />
+              <div
+                title="record new flashcard"
+                className="microphone"
+                data-linked-to="frontText"
+                onClick={this.handleRecordClick}
+              />
             </div>
             <div className="flashcard">
-              <div className="content">
-                <strong>back text</strong>
-                <textarea
-                  name="backText"
-                  id="backText"
-                  className="text"
-                  placeholder="Please press the record button"
-                  onChange={this.handleBackTextChange}
-                  value={this.props.backText}
-                />
-                <div
-                  title="record new flashcard"
-                  className="microphone"
-                  id="microphone-learning-2"
-                  data-linked-to="backText"
-                  data-id="2"
-                  onClick={this.handleRecordClick}
-                />
-              </div>
+              <strong>back text</strong>
+              <textarea
+                name="backText"
```

Comment:
I've the feeling, that this changes (in this file) are not extra existing.
maybe a merge-conflict issue?